http3/transport.go

```diff
@@ -276,6 +282,7 @@ func (t *Transport) getClient(ctx context.Context, hostname string, onlyCached b
 		}
 		select {
 		case <-cl.conn.HandshakeComplete():
+			traceTLSHandshakeDone(trace, cl.conn.ConnectionState().TLS, nil)
```

Comment:
This shouldn't be called for reused connections.

Reply:
removed